next.config.js

```diff
@@ -61,90 +60,84 @@ const hideWaypointTipContent = {
 	],
 }
 
-module.exports = withSwingset({
-	componentsRoot: 'src/components/**/*',
-	docsRoot: 'src/swingset-docs/*',
-})(
-	withHashicorp({
-		nextOptimizedImages: true,
-		css: false,
-		transpileModules: [
-			'@hashicorp/flight-icons',
-			/**
-			 * TODO: once Sentinel has been migrated into the dev-portal repository,
-			 * we should consider localizing the sentinel-embedded component. Should
-			 * first confirm with Cam Stitt that this component is not being used
-			 * elsewhere.
-			 */
-			'@hashicorp/sentinel-embedded',
-			'swingset',
-			'unist-util-visit',
-		],
-	})({
-		webpack(config) {
-			config.plugins.push(HashiConfigPlugin())
-			return config
-		},
-		async headers() {
-			return [temporary_hideDocsPaths, hideWaypointTipContent]
-		},
-		async redirects() {
-			const { simpleRedirects, globRedirects } = await redirectsConfig()
+module.exports = withHashicorp({
+	nextOptimizedImages: true,
+	css: false,
+})({
+	transpilePackages: [
+		'@hashicorp/flight-icons',
+		/**
+		 * TODO: once Sentinel has been migrated into the dev-portal repository,
+		 * we should consider localizing the sentinel-embedded component. Should
+		 * first confirm with Cam Stitt that this component is not being used
+		 * elsewhere.
+		 */
+		'@hashicorp/sentinel-embedded',
+		'unist-util-visit',
+	],
+	webpack(config) {
+		config.plugins.push(HashiConfigPlugin())
+		return config
+	},
+	async headers() {
+		return [temporary_hideDocsPaths, hideWaypointTipContent]
+	},
+	async redirects() {
```

Comment:
@dstaley when you have the time can you walk me through the redirection changes here? I'm a bit too new to completely understand these changes and would love to talk through them to make sure that we are not changing them in any meaningful way.

Reply:
Nothing about the redirects/rewrites has changed here. GitHub is just confused since the indentation has changed by removing the wrapping `withSwingset` call.